README.md

```diff
@@ -18,7 +18,6 @@ or CSS3 selectors.
 
 [![Concourse CI](https://ci.nokogiri.org/api/v1/teams/nokogiri-core/pipelines/nokogiri/jobs/ruby-2.4-system/badge)](https://ci.nokogiri.org/teams/nokogiri-core/pipelines/nokogiri?groups=master)
 [![Code Climate](https://codeclimate.com/github/sparklemotion/nokogiri.svg)](https://codeclimate.com/github/sparklemotion/nokogiri)
-[![Version Eye](https://www.versioneye.com/ruby/nokogiri/badge.png)](https://www.versioneye.com/ruby/nokogiri)
```

Comment:
I've got SSL certificate error on this url.

![2018-11-02 14 59 51](https://user-images.githubusercontent.com/167012/47896257-f83a1800-deaf-11e8-872d-aa61f23175f1.png)